rasa/shared/importers/autoconfig.py

```diff
@@ -52,7 +52,7 @@ def get_configuration(
         logger.debug("No configuration file was provided to the TrainingDataImporter.")
         return {}
 
-    config = rasa.shared.utils.io.read_config_file(config_file_path)
+    config = rasa.shared.utils.io.read_model_configuration(config_file_path)
```

Comment:
oh wow, so it was reading the wrong file?

Reply:
yeah, it validated using the wrong schema 🙈